source/vibe/http/websockets.d

```diff
@@ -697,108 +727,234 @@ final class IncomingWebSocketMessage : InputStream {
 	}
 }
 
+/// Magic string defined by the RFC for challenging the server during upgrade
+private static immutable s_webSocketGuid = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
```

Comment:
I always thought that this will always happen for `immutable` literals (at the least for string literals, not sure about arrays).